spec/queue.js

```diff
@@ -0,0 +1,76 @@
+import chai, { expect, assert } from 'chai'
+import chaiChange from 'chai-change'
+import Queue from '../src/queue'
+
+chai.use(chaiChange)
+
+describe('Queue', () => {
+  'use strict'
```

Comment:
Out of curiosity, why do you use `use strict` here, instead of at the top of the file?